JavaScript/JS-Week_1/Javascript_Homework/Housey Pricey/script.js

```diff
@@ -0,0 +1,31 @@
+// Peter's parameters
+let pWidth = 8;
+let pHeight = 10;
+let pDepth = 10;
+let pGardenSize = 100;
+let pRealPrice = 2500000;
```

Comment:
All those could be defined as `const`. Sorry for repeating the same comment.
The naming of variables can be improved upon. It's okay to have long descriptive variables, it's even preferred than abbreviations that might make sense only to you, at the period of time that you write the code. `peterHouseWidth` might be for example more clear than `pWidth`. I don't want to you to change anything based on this comment, but just to raise a thought about it. Naming things is very personal and subjective - what might make sense to one person, might be confusing to another.

Reply:
Actually I thought if I use let , it won't show any error. That's why I used it. But I understand now. Thank you for your valuable advice.